core/spree_core.gemspec

```diff
@@ -16,26 +16,4 @@ Gem::Specification.new do |s|
 
   s.files        = Dir['LICENSE', 'README.md', 'app/**/*', 'config/**/*', 'lib/**/*', 'db/**/*', 'vendor/**/*']
   s.require_path = 'lib'
-
-  s.add_dependency 'activemerchant', '1.78.0' # bumped to 1.78.0 to be compatible with OFN
-  s.add_dependency 'acts_as_list', '= 0.2.0'
-  s.add_dependency 'awesome_nested_set', '~> 3.0.0.rc.1'
-  s.add_dependency 'aws-sdk', '1.11.1' # temporarily locked down due to https://github.com/aws/aws-sdk-ruby/issues/273
-  s.add_dependency 'cancan', '~> 1.6.10'
-  s.add_dependency 'ffaker', '~> 1.16'
-  s.add_dependency 'highline', '= 1.6.18' # Necessary for the install generator
-  s.add_dependency 'httparty', '~> 0.11' # For checking alerts.
-  s.add_dependency 'json', '>= 1.7.7'
-  s.add_dependency 'kaminari', '~> 0.14.1'
-  s.add_dependency 'money', '5.1.1'
-  s.add_dependency 'paperclip', '~> 3.4.1'
-  s.add_dependency 'paranoia', '~> 2.0'
-  s.add_dependency 'rails', '~> 4.0.0'
```

Comment:
This basically means that spree_core is no longer holding us from upgrading rails!